DataCollector/DoctrineDataCollector.php

```diff
@@ -73,10 +73,7 @@ public function __construct(ManagerRegistry $registry, bool $shouldValidateSchem
         }
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function collect(Request $request, Response $response, ?Throwable $exception = null)
+    public function collect(Request $request, Response $response, ?Throwable $exception = null): void
```

Comment:
is this actually a BC break if someone extended this class (for whatever reason)? :thinking:

Reply:
Yes, why did you decide to do it? I mean I am not opposed, we don't have same BC promise as Symfony, but it's strange that this return type was adding seemingly arbitrarily while it wasn't done in other methods.

Reply:
Yeah did it just to fix the reported deprecation. I can also revert this change then

Reply:
Ok then let's keep it as I think the chances are low that someone extended this class